frontend/src/app/globals.css

```
@@ -76,13 +74,87 @@
-* {
+/* ── Base reset ──
+   Explicit reset replacing Tailwind's preflight (Tailwind's utilities were never
+   used — only preflight was in play). Mirrors the preflight rules this app relied
+   on so behavior is unchanged after dropping the dependency. */
+*,
+::before,
+::after {
   box-sizing: border-box;
+  border-width: 0;
+  border-style: solid;
+  border-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)